st2client/st2client/commands/resource.py

```diff
@@ -154,6 +155,7 @@ def get_resource_by_pk(self, pk, **kwargs):
         try:
             instance = self.manager.get_by_id(pk, **kwargs)
         except Exception as e:
+            traceback.print_exc()
```

Comment:
Is this intentional or you just added it for debugging only purposes?

If it's intentional, we should probably be consistent with the rest of the code and only print it if `--debug` flag is passed to the CLI.